kuma/payments/jinja2/payments/terms.html

```diff
@@ -0,0 +1,60 @@
+{% extends 'base.html' %}
+
+{% block title %}{{ page_title(_('MDN Payment terms')) }}{% endblock %}
+
+{% block site_meta %}
+    {{ super() }}
+    {% set seo_description = _('MDN Payment terms') %}
+    {% set social_title = _('MDN Payment terms') %}
+    <meta property="og:description" content="{{ seo_description }}">
+    <meta name="description" content="{{ seo_description }}">
+    <meta name="twitter:description" content="{{ seo_description }}">
+    <meta name="twitter:title" content="{{ social_title }}">
+    <meta property="og:title" content="{{ social_title }}">
+{% endblock %}
+
+
+{% block content %}
+    <div>
+        <h1>{{ _('Payment terms') }}</h1>
+        <br>
+        <p>
+            {{ _('Mozilla (that’s us) operates MDN Web Docs') }}
+        </p>
+        <p>
+            {% trans url="https://www.mozilla.org/en-US/about/legal/terms/mozilla/" %}
+                These Payment Terms cover your payment to MDN. <a href="{{ url }}">The Websites & Communications Terms of Use</a>
+                cover the MDN service itself. If there is any inconsistency between these terms and the
+                Websites & Communications Terms of Use, these Payment Terms apply.
+            {% endtrans %}
+        </p>
+        <h2>{{ _('Payment Authorization') }}</h2>
+        <p>
+            {% trans terms_url="https://help.github.com/articles/github-terms-of-service/", privacy_policy_url="https://help.github.com/articles/github-privacy-statement/" %}
+                You can support MDN with a one-time payment, or you can choose to schedule monthly payments. Your payments are not tax deductible.
```

Comment:
These should be separate paragraphs, wrapped in ``<p>`` tags.  If we retain the translation blocks, each paragraph should have its own ``{% trans %}`` block inside the ``<p></p>`` tags.

Reply:
fixed